public/theme/boltcms-2021/featureslist.twig

```diff
@@ -0,0 +1,108 @@
+{% extends 'partials/_master.twig' %}
+
+{% block main_container %}
+    <div class="subheader">
+        <h1 class="drop-left">Features for
+            <div class="slidingVertical">
+                <span>developers</span>
+                <span>designers</span>
+                <span>content creators</span>
+                <span>you</span>
+            </div>
+        </h1>
+        <div class="button-row">
+            {% include "partials/_features_buttons.twig" %}
+        </div>
+    </div>
+    {# {% set current = app.request.requestUri|split('/')|last %} #}
```

Comment:
Let's get rid of the commented out code, if we don't need it :-)

Reply:
Check